Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -125,7 +125,7 @@ class State(enum.Enum):
 
 
 class OWTestLearners(OWWidget):
-    name = "Test & Score"
+    name = "Test and Score"
```

Comment:
This changes the name of the widget in Orange. Which means all schemas that had "Test & Score" before could fail. Please check.

Reply:
It works (I checked on an example workflow).